src/scripts/parseCodePath.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Since we don't have eslint in this repo, what are you thoughts on removing this line? Or do you want to keep it the exact same as it was in the previous repo?

Reply:
I was thinking about whether or not we did want to include eslint in this repo. Any preferences there? If not, happy to remove.

Reply:
No preference! I feel like maybe in the future if we end up developing scripts within this repo, but for now I don't think we need it.